File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.java
class Solution {
    public int solution(String ineq, String eq, int n, int m) {
        return n == m && eq.equals("=") ? 1 
            : n < m && ineq.equals("<") ? 1
            : n > m && ineq.equals(">") ? 1
            : 0;
    }
}

File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.py
def solution(ineq, eq, n, m):
    op = ineq+eq
    if(op == ">="):
        return int(n>=m)
    elif(op == "<="):
        return int(n<=m)
    elif(op == ">!"):
        return int(n>m)
    elif(op == "<!"):
        return int(n<m)
    return 0